rolemanager/role_manager.go

```diff
@@ -271,6 +271,21 @@ func (m *mongoBackedRoleManager) DeleteScope(scope gimlet.Scope) error {
 	return m.retryTransaction(updateFunc)
 }
 
+func (m *mongoBackedRoleManager) GetScope(id string) (*gimlet.Scope, error) {
+	var err error
+	scopeCollection := m.client.Database(m.db).Collection(m.scopeColl)
+	result := scopeCollection.FindOne(context.Background(), bson.M{"_id": id})
```

Comment:
why not require contexts as arguments? 

another alternative, perhaps better is to do a context.WithCancel(context.TODO())? 

should we stash a context in the manager?

Reply:
Ah good point, the methods should have contexts as args. I added it here and will do so going forward. We want the caller to be able to cancel these with the granularity of a specific operation, so I think this makes sense